### PR TITLE
Updated ReactiveUI_Mac64 to output to Xamarin.Mac folder.

### PR DESCRIPTION
**Pull request by mteper**
Not sure what else to update to have proper things done for NuGet, but at least now the unified mac build won't overwrite the MonoMac version.

### CODE REVIEW

File: ReactiveUI/ReactiveUI_Mac64.csproj [hunk 1]
```diff
@@ -17,7 +17,7 @@
     <DebugSymbols>True</DebugSymbols>
     <DebugType>full</DebugType>
     <Optimize>False</Optimize>
-    <OutputPath>bin\Debug\MonoMac</OutputPath>
+    <OutputPath>bin\Debug\Xamarin.Mac</OutputPath>
     <IntermediateOutputPath>obj\Debug\MonoMac</IntermediateOutputPath>
```
Comment (paulcbetts) [inline ReactiveUI/ReactiveUI_Mac64.csproj]: Fix these too

**Comment by paulcbetts** [comment]
You also need to create templates in the `NuGet` folder (just empty files), as well as patch `MakeRelease.ps1` to have the extra platform

**Comment by mteper** [comment]
I think we have to start in Splat, no?

**Comment by paulcbetts** [comment]
> I think we have to start in Splat, no?

Splat already has 64-bit Xamarin.Mac support afaik, we just need to make sure the minversion in our release is >= the version where it was added

**Comment by mteper** [comment]
We are depending on Splat 1.6.0, I think. At least that's what the HintPath in the .xcodeproj file points to. As far as I can tell, that version included the Unified Xamarin.Mac binaries.

**Pull request merged**